extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -272,28 +271,40 @@ export abstract class AzureAuth implements vscode.Disposable {
 	}
 
 	protected async makePostRequest(uri: string, postData: { [key: string]: string }, validateStatus = false) {
-		const config: AxiosRequestConfig = {
-			headers: {
-				'Content-Type': 'application/x-www-form-urlencoded'
+		try {
+			const config: AxiosRequestConfig = {
+				headers: {
+					'Content-Type': 'application/x-www-form-urlencoded'
+				}
+			};
+
+			if (validateStatus) {
+				config.validateStatus = () => true;
 			}
-		};
 
-		if (validateStatus) {
-			config.validateStatus = () => true;
+			return await axios.post(uri, qs.stringify(postData), config);
+		} catch (ex) {
+			console.log(JSON.stringify(ex?.response?.data, undefined, 2));
```

Comment:
Considering putting a message prefix like `Unexpected error making Azure auth request : ` so we can easily tell where it's coming from.

(for this and the other ones below)

Reply:
I'll take this advice and just modify a bit and mention the code point to make it a little more obvious :)